### PR TITLE
Add crud

### PR DESCRIPTION
**Pull request by IrynaMatviichuk**

### CODE REVIEW

File: kubehub/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CloudProvider',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('cp_type', models.CharField(max_length=20)),
```
Comment (kopachevsky) [inline kubehub/migrations/0001_initial.py:18]: why cp_type, is type reserved keyword?

File: kubehub/serializers.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        fields = ('id', 'cp_type', 'name', 'api_endpoint', 'password')
+
+
+    def create(self, validated_data):
+        cloud_probider = CloudProvider.objects.create(**validated_data)
+        return cloud_probider
+
+
+    def update(self, instance, validated_data):
+        instance.cp_type = validated_data.get('cp_type', instance.cp_type)
```
Comment (kopachevsky) [inline kubehub/serializers.py:17]: can we add validation as enum? like "PROX_MOX", "AWS"

**Review by kopachevsky — APPROVED** [review]

File: kubehub/migrations/0001_initial.py (hunk 1 above)
Comment (IrynaMatviichuk) [reply kubehub/migrations/0001_initial.py:18]: Yes, `type()` function returns class type

File: kubehub/serializers.py (hunk 2 above)
Comment (IrynaMatviichuk) [reply kubehub/serializers.py:17]: Yes, I just wanted to know full list of all providers types

Comment (IrynaMatviichuk) [reply kubehub/serializers.py:17]: Fixed 8048aae

File: kubehub/migrations/0001_initial.py (hunk 1 above)
Comment (kopachevsky) [reply kubehub/migrations/0001_initial.py:18]: ok!

**Pull request merged**